src/embed_tests/StateSerialization/MethodSerialization.cs

```diff
@@ -33,6 +33,10 @@ static T SerializationRoundtrip<T>(T item)
     {
         using var buf = new MemoryStream();
         var formatter = RuntimeData.CreateFormatter();
+
+        if (formatter == null)
```

Comment:
It might be a good idea to make the whole test class `[Obsolete]` to prevent warnings

Reply:
Obsolete code warning that comes from using `BinaryFormatter`. You can see it on GitHub review page.